### DISCUSSION TITLE
How to custom Message Text Press event?

### DISCUSSION THREAD

**Discussion by cleandevcode** [GetStream/stream-chat-react-native#1282] [Q&A]
My tech stack:
`stream-chat`: `6.1.0`
`stream-chat-expo`: `^4.1.0`
`react`: `16.15.1`
`expo`: `^40.0.0`
`react-natve`: `0.63.4`
By default, pressing message text which is detected as a link will open the web browser in the mobile app.

By the way are there any option to catch the pressing event?

![image](https://user-images.githubusercontent.com/52365167/163174947-554f0bd4-9fd0-4356-a25a-8be116179d12.png)

Thanks.

**Comment by evanserickson** [comment]
Bump. Did you find a solution?

By the way, thanks for the question because you answered my question, "Can you use expo with it" and since you are using expo I assume we can.

I also asked them to update docs to make it more clear that you can use expo.
